README.md

```diff
@@ -1,200 +1,35 @@
-`more-info/_index.md > node/_index.md > pokt/_index.md`.
-
-### Adding new API Docs
-
-API Docs are a unique type of content that live in the `content/api-docs/`
-directory. To create a new API Docs page, we'll need a URL that points to an
-[OAS 3.0] or newer specification we intend on updating and regenerating when
-changes occur, we can make use of the `static/oas/` directory. If not, we can
-simply use the URL of where the document is hosted remotely.
-
-The hard part of Adding a new API is building the specification, once that's
-done, there's only 2 things we'll need to ask about the current specification.
-
-1. Do we need to ignore the paths in the spec file?
-2. Is there a path that all of the API endpoints use that's different from the server?
-
-#### JSON RPC vs OAS
-
-The first case would happen with any JSON-RPC API that sends all of its
-requests to the same path. [OAS 3.0] different methods will happen on different
-paths, so it doesn't 100% support describing these kinds of API. However, you
-can force a JSON-RPC API into an [OAS 3.0] specification by using the method
-names as the unique paths. Doing this will mean that any client/server codegen
-from the specification will be wrong, but we're using these specifications to
-enable interactive, in-browser documentation tooling, so this kind of breakage
-will be acceptable for our case. It's through this process that we'll be able
-to get the [Ethereum JSON-RPC
-API](https://ethereum.org/en/developers/docs/apis/json-rpc/) into our
-interactive documentation tooling.
-
-#### A Different Shared Path
-
-This second case only occurs if the first case also happens, but this would
-occur if shared URL for the API Spec is different from the server URL. The
-example of this that comes to mind is Avalanches C-Chain manages all EVM
-compatibility through a `/ext/bc/C/` suffix to the URL. So the C-Chain
-specification would hold for both of these cases.
-
-#### Adding the New Page
-
-Once the following have been determined, a new API Doc page can be created,
-that can be defined entirely in the frontmatter. We'll use an example that
-doesn't have the shared path issue, and one that does have a shared path,
-and a non-server URL to show the example of adding both.
-
-The following would be saved as `content/api-docs/pokt.md` to enable
-the Pocket Mainnet API docs.
-
-```yaml
----
-type: api-doc
-name: Pocket Mainnet
-spec_url: /oas/pokt.json # Hosted in the /static directory.
-portal_prefix: mainnet
----
-```
-
-The following would be saved as `content/api-docs/avalanche-c-chain.md` to
-enable the Avalanche C-Chain API docs.
-
-```yaml
----
-type: api-doc
-name: Avalanche C-Chain
-spec_url: https://path-to-avax/spec.json
-portal_prefix: avax-mainnet
-ignore_api_path: true # Warning: explicitly setting to false still ignores path
-api_slug: /ext/bc/C/
----
-```
-
-## Customizing Content
+4. View the built docs in your browser at http://localhost:1131.
```

Comment:
Should this be localhost:1313? At least that's what mine has always defaulted to.

Reply:
Great catch, yes, that's incorrect.

Reply:
Fixed now.